lib/team.rb

```diff
@@ -1,11 +1,9 @@
 class Team
-  attr_reader :team_id,
-              :franchiseid,
-              :teamname
+  attr_reader :team_id, :franchise_id, :teamname
 
   def initialize(attributes)
     @team_id = attributes[:team_id].to_i
-    @franchiseid = attributes[:franchiseid].to_i
+    @franchise_id = attributes[:franchiseid].to_i
     @teamname = attributes[:teamname]
```

Comment:
I like changing franchise_id  so we can call it with conventional syntax later. We can look into doing that with teamname (team_name) if we'd like to sort out some of the existing methods which call teamname.